File: documentation/user/en/solve/examples/filtering-products-in-category/breadcrumb-category.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Category"),
        	FilterBy(
        		AttributeEquals("url", "/en/smartwatches"),
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en"))
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "name"),
        			HierarchyContent(
        				EntityFetch(
        					AttributeContent("code", "name", "level")
        				)
        			)
        		)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/filtering-products-in-category/breadcrumb-category.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Category"),
				filterBy(
					attributeEquals("url", "/en/smartwatches"),
					entityLocaleEquals(Locale.forLanguageTag("en"))
				),
				require(
					entityFetch(
						attributeContent("code", "name"),
						hierarchyContent(
							entityFetch(
								attributeContent("code", "name", "level")
							)
						)
					)
				)
			)
		);
	}
);
